apiserver/facades/client/application/application.go

```diff
@@ -76,7 +76,6 @@ type APIBase struct {
 	check                     BlockChecker
 	repoDeploy                DeployFromRepository
 
-	model              Model
 	modelInfo          model.ModelInfo
```

Comment:
I think the only information that is really needed is that of the model type at the moment.

Reply:
Hi tom, the modelUUID is required as well. Would it be better to just pass both the uuid and type as separate fields rather than passing the entire model? I'm not too sure if other model fields would be required in the future.